CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 ## [Unreleased]
+
+## [v0.0.12-alpha] - 2022-12-7
+- Dynamic Contract C++ Bindings : Encode,Call,Send
+- Minting C++ Example : Encode,Send
```

Comment:
Changes on play-cpp-sdk itself are needed to add:
- Replace custom cxx-build with official cxx-build
- Added the walletconnect registry API support
- Add app token for sepgrep CI
- Fix Walletconnect request deadlock issue
- Upgrade ethers to 1.0